sass/cd/cd-header/_cd-global-header.scss

```diff
@@ -5,13 +5,49 @@
 .cd-global-header {
   .cd-global-header__actions {
     float: right;
+
+    [dir="rtl"] & {
+      float: left;
+    }
   }
 
   &__language-switcher {
     float: left;
+
+    [dir="rtl"] & {
+      float: right;
+    }
   }
 
   &__user-menu {
     float: left;
+
+    [dir="rtl"] & {
+      float: right;
+    }
+  }
+}
+
+// If there are multiple menus, separate them with pipes and margins */
+/* This should correspond with the dropdown menu breakpoint */
+.js {
+  .cd-global-header__actions > * + nav {
```

Comment:
The wildcard selector is because the Language Switcher element is a `div` (Drupal block) whereas the other elements in `.cd-global-header__actions` are `nav` (Drupal Menu blocks).